assets/navigations/components/Navigations.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import EditNavigation from './EditNavigation';
+import NavigationList from './NavigationList';
+import SearchResultsInfo from 'wire/components/SearchResultsInfo';
+import {setError} from '../actions';
+import {gettext} from 'utils';
+
+class Navigations extends React.Component {
```

Comment:
given that this is connected component would be better for encapsulation if you use map state/dispatch to props here rather than on the NavigationsApp which is only passing it all down without having any state